frontend/lib/norent/footer.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { NorentRoutes as Routes } from "./routes";
+import { Link } from "react-router-dom";
+
+const MAILCHIMP_URL =
+  "https://nyc.us13.list-manage.com/subscribe/post?u=d4f5d1addd4357eb77c3f8a99&amp;id=588f6c6ef4";
+
+const EmailSignupForm = () => (
```

Comment:
@toolness I have a feeling we can reuse some of your form components to make this more accessible. Any suggestions?